src/Acuminator/Acuminator.Utilities/Roslyn/Semantic/Symbols/Attributes/DataTypeAttributeSymbols.cs

```diff
@@ -2,13 +2,75 @@
 
 using Microsoft.CodeAnalysis;
 using Acuminator.Utilities.Roslyn.Constants;
+using System.Collections.Immutable;
+using System.Collections.Generic;
 
 namespace Acuminator.Utilities.Roslyn.Semantic.Symbols
 {
 	public class DataTypeAttributeSymbols : SymbolsSetBase
 	{
 		internal DataTypeAttributeSymbols(Compilation compilation) : base(compilation)
-		{ }
+		{
+			var types = new List<INamedTypeSymbol?>() {
+				PXDBStringAttribute,
+				PXStringAttribute,
+				PXDBBinaryAttribute,
+				PXDBDecimalAttribute,
+				PXDecimalAttribute,
+				PXDBDoubleAttribute,
+				PXDoubleAttribute,
+				PXDBFloatAttribute,
+				PXFloatAttribute,
+			};
+
+			var unboundTypes = new List<INamedTypeSymbol?>() {  
+				PXLongAttribute,
+				PXIntAttribute,
+				PXShortAttribute,
+				PXStringAttribute,
+				PXByteAttribute,
+				PXDecimalAttribute,
+				PXDoubleAttribute,
+				PXFloatAttribute,
+				PXDateAttribute,
+				PXGuidAttribute,
+				PXBoolAttribute,
+				PXVariantAttribute,
+			};
+
+			DataAttributesWithLength = types.ToImmutableArray()!;
+			UnboundDataTypeAttributes = unboundTypes.ToImmutableArray()!;
+
+			var guidTypeSymbol = Compilation.GetTypeByMetadataName(typeof(System.Guid).FullName)!;
+
+			TypeAttributesToCLRTypesMap = new Dictionary<ITypeSymbol, ITypeSymbol>(SymbolEqualityComparer.Default)
```

Comment:
@acasj there already exists map between attributes and CLR types: `Acuminator.Utilities.Roslyn.PXFieldAttributes.FieldTypeAttributesMetadataProvider`
It should be reused.